img_topic : /usb_cam/image_raw
broadcast_topic : /safeguard/broadcast_info
debug_mode : True